OracleWebLogic/samples/12212-domain/container-scripts/commonfuncs.py

```diff
@@ -6,13 +6,24 @@
 # Environment Vars
 hostname       = socket.gethostname()
 # Admin Vars
+admin_name     = os.environ.get('ADMIN_NAME', 'AdminServer')
 admin_username = os.environ.get('ADMIN_USERNAME', 'weblogic')
-admin_password = os.environ.get('ADMIN_PASSWORD') # this is read only once when creating domain (during docker image build)
+admin_password = "ADMIN_PASSWORD"
```

Comment:
Where does admin_password variable gets the actual value? Here, it will be hardcoded to "ADMIN_PASSWORD"